### ISSUE TITLE
tlf-primary-pilot3.rtf : Titles, Footnotes, and timestamp missing

### ISSUE THREAD

**Issue by laxamanaj** [RConsortium/submissions-pilot3-adam#109]
Hi, @kaz462 .

As I was working on #107 I noticed that tlf-primary-pilot3.rtf was missing Titles, footnotes and Timestamp.  When I run it currently I see this when I open the .rtf : 
![image](https://github.com/RConsortium/submissions-pilot3-adam/assets/95468392/d000b422-d233-44f5-bb1a-4eb5202b654e)

When I go to the output in Pilot 1, it looks like this : https://github.com/RConsortium/submissions-pilot1-to-fda/blob/main/m1/us/report-tlf.pdf
![image](https://github.com/RConsortium/submissions-pilot3-adam/assets/95468392/078e13f1-ccd7-4111-84fa-8ff66ab3c43c)

Can you take a look at this?  Thanks.
Joel

Linked to : #39 #40

**Comment by laxamanaj** [comment]
Hi, @kaz462 .  Maybe I'm missing something.  How are you opening the output?

When I click on the .rtf in the output folder, it downloads the .rtf file to my local computer, but when I open it, it shows the table that I see in my first comment of this issue.

**Comment by laxamanaj** [comment]
After meeting with @kaz462 (Thanks, Kaz!!!), we found out that it was just a software issue on my end, where I had to convert the file into a more readable format to see the titles and footnotes.  This is now my current view after Kaz helped me investigate.  This issue can now be closed.

![image](https://github.com/RConsortium/submissions-pilot3-adam/assets/95468392/8c0cc96a-4757-4b1f-b39c-f78cbb76ea0c)
